base/include/SoundRecord.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "Module.h"
+
+class SoundRecordProps : public ModuleProps
+{
+public:
+    SoundRecordProps(
+        int _sampleRate,
+        int _channel,
+        int _device,
+        int _proccessingRate) : sampleRate(_sampleRate),
+                                channel(_channel),
+                                device(_device),
+                                proccessingRate(_proccessingRate)
```

Comment:
Is this really processing rate ? I saw ithe test sets it to 200 and it looks like it is a time interval in milliseconds. In general, rate = (time interval )^-1 i.e. inverse. So let's use a better name to avoid confusing the audience. I think it is "processingInterval" in SFML, we can use "processingIntervalMS" to be clear.